unstructured/errors.py

```diff
@@ -0,0 +1,8 @@
+class PdfMaxPagesExceededError(ValueError):
+    """Error raised, when number of PDF pages exceeds max_pages limit
+    and HI_RES strategy is chosen.
+    """
+
+    def __init__(self, message):
+        self.message = message
```

Comment:
we don't need keeping the message in this class. So the whole `__init__` method is not needed.